callouts/python/extproc/example/basic_callout_server.py

```diff
@@ -57,7 +57,7 @@ def on_request_body(self, body: HttpBody, _) -> BodyResponse:
   def on_response_body(self, body: HttpBody, _) -> BodyResponse:
     """Custom processor on the response body."""
     logging.debug("Received response body callout: %s", body)
-    return add_body_mutation(body='new-body', clear_body=True)
+    return add_body_mutation(clear_body=True)
```

Comment:
why does the basic callout server need to touch body or clear the response?

Reply:
This is intended just as an example server with an assortment of possible interactions. I can reduce the complexity to make it more of a basic example.